packages/manager/src/features/Linodes/LinodeCreate/AdditionalOptions/AdditionalOptions.tsx

```diff
@@ -26,7 +26,7 @@ export const AdditionalOptions = () => {
   });
 
   const showAlerts =
-    flags.aclpBetaServices?.alerts && isAclpAlertsSupportedRegionLinode;
+    aclpBetaServices?.['linode']?.alerts && isAclpAlertsSupportedRegionLinode;
```

Comment:
```suggestion
    aclpBetaServices?.linode?.alerts && isAclpAlertsSupportedRegionLinode;
```
nit: This isn't very clean or ideal unless the key is dynamic, invalid, or not accessible via dot notation

We should use one of the following for better readability: 
- `flags.aclpBetaServices?.linode?.alerts;` (clean and readable)
OR
- `aclpBetaServices?.linode?.alerts;` (also clean, but might lose some clarity)

Reply:
I've updated, thanks for pointing out